ext4/tar2ext4/tar2ext4.go

```diff
@@ -312,3 +272,15 @@ func ConvertAndComputeRootDigest(r io.Reader) (string, error) {
 	hash := dmverity.RootHash(tree)
 	return fmt.Sprintf("%x", hash), nil
 }
+
+// ConvertToVhd converts given io.ReadWriteSeeker to VHD, by appending the VHD footer with a fixed size.
+func ConvertToVhd(w io.ReadWriteSeeker) error {
```

Comment:
This could just take in a `io.WriteSeeker` right? We don't need w to implement `Read`